Libraries/LibGfx/Painter.h

```diff
@@ -24,15 +24,15 @@ class Painter {
     virtual void clear_rect(Gfx::FloatRect const&, Gfx::Color) = 0;
     virtual void fill_rect(Gfx::FloatRect const&, Gfx::Color) = 0;
 
-    virtual void draw_bitmap(Gfx::FloatRect const& dst_rect, Gfx::ImmutableBitmap const& src_bitmap, Gfx::IntRect const& src_rect, Gfx::ScalingMode, Span<Gfx::Filter> filters, float global_alpha) = 0;
+    virtual void draw_bitmap(Gfx::FloatRect const& dst_rect, Gfx::ImmutableBitmap const& src_bitmap, Gfx::IntRect const& src_rect, Gfx::ScalingMode, ReadonlySpan<Gfx::Filter> filters, float global_alpha) = 0;
```

Comment:
if we know that `filters` argument has to be `ReadonlySpan<Gfx::Filter>` then could you make it so in the change where this parameter is introduced?

Reply:
Done :)